client/src/App.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
```

Comment:
Don't push any edits in the App component either jsx or css. I think we agreed on that.
Only use the App component to test your work and eventually it will contain the app once we're finished of the parts.

Reply:
Sorry :sweat_smile: